src/app/screens/Receive.tsx

```diff
@@ -32,6 +34,19 @@ function Receive() {
     });
   }
 
+  useEffect(() => {
+    if (invoice) {
+      const pollForPayment = poll({
+        fn: () => utils.call("checkPayment", { paymentHash: invoice.rHash }),
+        validate: (payment) => payment.paid,
+        interval: 5000,
+        maxAttempts: 10,
```

Comment:
I think we should poll a little longer but decrease the interval a bit (maybe 3 seconds? - 5 seconds feels quiet long?)